README.md

```diff
@@ -5,20 +5,20 @@
 > of it, as a proof of concept. Feel free to play around with it, but be
 > cautious. Examine the code.
 
-AMP Packager creates "AMP Packages" (implemented as [Signed HTTP
-Exchanges](https://tools.ietf.org/html/draft-yasskin-httpbis-origin-signed-exchanges-impl-00))
-containing AMP documents. These packages are consumed by the [Google AMP
-Cache](https://www.ampproject.org/docs/fundamentals/how_cached) and cached.
-Eventually, a future variant of these packages will be linked to from Google
-Search instead of normal AMP page. The packages are signed with a certificate,
-which means that Chrome users will see that certificate's domain in the URL bar
-instead of `google.com` or `ampproject.org`, and that the web page will run on
-that origin.
-
-These packages have a maximum lifetime of 7 days, which minimizes the risk of
-another cache serving a stale copy of this signed content. In the future, the
-[AMP update-cache API](https://developers.google.com/amp/cache/update-cache)
-will support updating AMP Packages, though it doesn't yet.
+AMP Packager is a tool to [improve AMP
+URLs](https://amphtml.wordpress.com/2018/01/09/improving-urls-for-amp-pages/).
+By running it in a proper configuration, web publishers may (eventually) have
+origin URLs appear in AMP search results.
+
+It works by creating [Signed HTTP
+Exchanges (SXGs)](https://tools.ietf.org/html/draft-yasskin-httpbis-origin-signed-exchanges-impl-00))
+containing AMP documents, signed with a certificate associated with the origin,
+with a maximum lifetime of 7 days. In the near future, the [Google AMP
+Cache](https://www.ampproject.org/docs/fundamentals/how_cached) will ingest,
+cache, and serve them, similar to how it does for normal AMP HTML documents.
+When a user loads such an SXG, Chrome validates the signature and then displays
+certificate's domain in the URL bar instead of `google.com`, and runs the web
```

Comment:
"runs the web page" sounds a bit awkward.... but i don't have a suggestion right now.